[Word] (Desktop only) Document WordApiHiddenDocument 1.3 requirement set (#1301)
OfficeDev/office-js-docs-reference: docs/docs-ref-autogen/overview/overview.md

The following is a list of APIs for the [supported Office host applications](/office/dev/add-ins/overview/office-add-in-availability). Common API links include all APIs not attributed to a particular host (as detailed in [Office Common API requirement set](/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets)). Other items link to a version of the API reference documentation for that host based on a requirement set. The reference documentation will be versioned to include all APIs up to and including that requirement set (for example, ExcelApi 1.3 shows APIs in ExcelApi 1.1, 1.2, 1.3 as well as the Common APIs).

## Special requirement sets

> The online-only and desktop-only requirement sets are special requirement sets. The online-only requirement set contains the latest APIs for the web version of the supporting applications (currently Excel and Word). The desktop-only requirement set contains the latest APIs for the Windows and Mac versions of the supporting applications (currently Word). However, those APIs may not yet be fully supported across all platforms. For more information, see one of the following as applicable.

Reviewer: This "however" sentence sounds a little strange with an additional sentence before it.
```suggestion
The online-only and desktop-only requirement sets are special requirement sets. The online-only requirement set contains the latest APIs for the web version of the supporting applications (currently Excel and Word). The desktop-only requirement set contains the latest APIs for the Windows and Mac versions of the supporting applications (currently Word). APIs in these requirement sets aren't fully supported across all platforms. For more information, see one of the following as applicable.
```
